[2024-03-11T13:56:55.741+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2021-08-03T00:00:00+00:00 [queued]>
[2024-03-11T13:56:55.806+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2021-08-03T00:00:00+00:00 [queued]>
[2024-03-11T13:56:55.809+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T13:56:56.191+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): second_task> on 2021-08-03 00:00:00+00:00
[2024-03-11T13:56:56.266+0000] {standard_task_runner.py:60} INFO - Started process 2281 to run task
[2024-03-11T13:56:56.334+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2021-08-03T00:00:00+00:00', '--job-id', '554', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp52ldy4ik']
[2024-03-11T13:56:56.361+0000] {standard_task_runner.py:88} INFO - Job 554: Subtask second_task
[2024-03-11T13:56:56.940+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2021-08-03T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T13:56:57.432+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-03T00:00:00+00:00'
[2024-03-11T13:56:57.436+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T13:56:57.445+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and I ran after task1']
[2024-03-11T13:56:57.508+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T13:56:57.551+0000] {subprocess.py:93} INFO - hey, I am task2 and I ran after task1
[2024-03-11T13:56:57.560+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T13:56:57.698+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20210803T000000, start_date=20240311T135655, end_date=20240311T135657
[2024-03-11T13:56:57.851+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T13:56:58.118+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
